### PR TITLE
EM ladder updates

### PR DESCRIPTION
**Pull request by Guldbek**
Still a work in progress.

The EM ladder is missing a more personal "touch" with their direct reports.

### CODE REVIEW

**Comment by mathias-abtion** [comment]
I definitely agree, and I think these are good points to add. Another one I thought about having under level 3 was "Understands the importance of listening." but I was not sure if it was perhaps a bit too much.

File: career/em-ladder.md [hunk 1]
```diff
@@ -54,6 +56,7 @@ client projects.
 - [ ] Empowers direct reports to own their work.
 - [ ] Ensures direct reports are aware of growth and learning opportunities and
   helps them leverage these.
+- [ ] Empowers direct reports to speak up about their pain points.
```
Comment (hcarreras) [inline career/em-ladder.md:59]: Isn't this one redundant?

**Review by hcarreras — COMMENTED** [review]
Good addition!

**Comment by hcarreras** [comment]
`Understands the importance of listening.`. In my experience, anything that starts with "understands" will be automatically checked by any individual filling in their career ladder :)

**Comment by hcarreras** [comment]
I've just realized that the name of the branch includes your initials. That's a usual practice in some places, but it does not play well with the collaborative/short-lived branches or pairing. So, it's better to avoid it.

**Comment by mathias-abtion** [comment]
@hcarreras: I think you have a good point regarding the "Understands the importance of listening" statement, and if we want it to be mentioned it should probably be in the introduction text instead of as a point under one of the levels.

**Comment by Guldbek** [comment]
> I've just realized that the name of the branch includes your initials. That's a usual practice in some places, but it does not play well with the collaborative/short-lived branches or pairing. So, it's better to avoid it.

Yeah, understandable! :)

File: career/em-ladder.md (hunk 1 above)
Comment (Guldbek) [reply career/em-ladder.md:59]: @hcarreras 

With the previous one added?

Comment (hcarreras) [reply career/em-ladder.md:59]: Yeap. I was wondering what's the difference between those 2 points? Or in which scenario a person would only mark one of them (or be working on only of the 2).

Comment (Guldbek) [reply career/em-ladder.md:59]: This first scenario is where the EM, is trying to identify their direct reports pain points.
"X is clearly stressed, The EM can see by X's harvest they are jumping projects 7 times a day" 

Where the other one would be encourage their direct reports to speak up about their pain points.
I know it's quite common for junior devs, to be hesitant to speak up when they have pain points, which is why i think there is quite the difference :)

Please let me hear your thoughts.

**Comment by Guldbek** [comment]
Closed this one due to the initials in branch name.

I've merged the changes in a different branch.

**Pull request closed without merging**